src/generator/operations.ts

```diff
@@ -245,7 +245,7 @@ function formatHeaderResponseValue(propName: string, header: string, schema: Sch
 }
 
 function generateOperation(clientName: string, op: Operation, imports: ImportManager): string {
-  if (isPageableOperation(op) && op.language.go!.paging.member === op.language.go!.name) {
+  if (isPageableOperation(op) && op.language.go!.paging.member === op.language.go!.name || op.language.go!.name.startsWith('Next')) {
```

Comment:
I'm wondering how safe/reliable this name check is.  Did you figure out why the existing check doesn't work?  It works for the other methods.

Reply:
Yes I agree, I think the true fix might end up being in autorest. The problem is that the way nextOperationWithQueryParams is declared varies from the other member operations. However, I believe next link being null is purposeful for this operation: 

```
"x-ms-pageable": { "nextLinkName": "odata.nextLink", "itemName": "values", "operationName": "Paging_nextFragment" },
        "operationId": "Paging_nextFragment",
```

vs

```
 "x-ms-pageable": { "nextLinkName": null, "itemName": "values" },
        "operationId": "Paging_nextOperationWithQueryParams",
```
Simply adding operationName isn't enough for it to be marked as a member

Reply:
I see, I was looking at the wrong thing.

The existing heuristic I introduced is half-baked :p.  I think a better solution is during transformation find all the operation IDs that also appear in `x-ms-pageable.operationName` and flag them for exclusion.  What do you think?

Reply:
What change is needed in the swagger?

Reply:
Adding `x-ms-pageable.operationName`

Reply:
It's already there, see operation `Paging_getWithQueryParams`

Reply:
Ah ok, I think I hadn't fully grasped where you wanted to get the operationName from. Got it now

Reply:
Done